cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/queue/AbstractQueueSpecificationGenerator.java

```diff
@@ -72,7 +72,7 @@
       }
 
       if (connection.getSourceType() == FlowletConnection.Type.STREAM) {
-        builder.add(createSpec(QueueName.fromStream(outputName),
+        builder.add(createSpec(QueueName.fromStream(app.getNamespaceId(), outputName),
```

Comment:
So this is like a hack to represent Id.Stream with QueueName. I think it's ok for this PR to keep this PR small, but the proper change should be changing the `QueueSpecification` to carry the simple name (inside a Flow, we never need namespace). That requires some bigger refactoring that we can do it later. Maybe create a JIRA so that we don't lose track of it.